src/renderer/src/pages/home/index.tsx

```
@@ -1,16 +1,19 @@
-const Home = () => {
-  const ipcHandle = (): void => window.electron.ipcRenderer.send('ping')
+import Chart from '@renderer/components/home/Chart/Chart'
+import { flex } from '@renderer/utils'
+import { styled } from 'styled-components'
 
+const Home = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)